js/index.js

```diff
@@ -1 +1,8 @@
-//Implement the red light using jQuery. Don't forget to add the script tags.
\ No newline at end of file
+$(document).ready(function){
+  //Hide all answers when page is loaded
+  $("#answer1").hide();
```

Comment:
$('#goButton').click(greenLight);
something like this to add listeners for each buttons